### PR TITLE
dist: document IOTLAB_PROFILE variable fo IoT-Lab-support

### PR DESCRIPTION
**Pull request by miri64**
Did not know this variable was available up until now, so I thought adding it to the doc would be helpful for others, too.

### CODE REVIEW

File: dist/testbed-support/README.iotlab.md [hunk 1]
```diff
@@ -57,9 +58,9 @@ if `IOTLAB_EXP_ID` is not set.
 
 This schedules a new experiment on the FIT IoT-LAB and waits until it enters
 "Running" state. It will request `IOTLAB_NODES` nodes of type `IOTLAB_TYPE`
-for `IOTLAB_DURATION` minutes at site `IOTLAB_SITE`. With `IOTLAB_PHY_NODES`
-it is possible to choose specific nodes for this experiment by using the resource id
-string format as described above.
+for `IOTLAB_DURATION` minutes at site `IOTLAB_SITE` with `IOTLAB_PROFILE`
```
Comment (cgundogan) [inline dist/testbed-support/README.iotlab.md]: I would suggest to use "using" instead of "with", because the next sentence begins again with "WITH".

Comment (miri64) [inline dist/testbed-support/README.iotlab.md]: Done (sorry for the delay)

**Comment by miri64** [comment]
@cgundogan okay to squash?

**Comment by cgundogan** [comment]
yes :+1: please squash

**Comment by miri64** [comment]
Squashed

**Comment by cgundogan** [comment]
No need for Murdock here. ACK and GO

**Pull request merged**